.vscode/launch.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
The problem of having this config in github, is that it'll overwrite any custom launch configuration  that you may have locally.  It actually did that silently for me when I pulled this change. Each developer might have their own preferences on how to setup the launch configuration.
We have a section in mapsdk README.md with vscode settings: "Debugging in the browser from Visual Studio Code". Maybe it should better go there?

Reply:
I removed it and added a section to the README.md